### PR TITLE
RD-4825: Show token description as a part of DeleteModal content

### PR DESCRIPTION
**Pull request by NorbertAmartus**
## Description

Changes consists of adding a translation which could display the token description and providing it to the `<DeleteModal />` 

## Screenshots / Videos

N/A

## Checklist

- [x] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [ ] I verified that all tests and checks have passed.
- [ ] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [x] I added proper labels to this PR.

## Tests

 

N/A

## Documentation

N/A

### CODE REVIEW

File: widgets/tokens/src/RemoveTokenButton.tsx [hunk 1]
```diff
@@ -11,20 +11,27 @@ const { useBoolean } = Stage.Hooks;
...
+    token: TokensWidget.DataItem;
     toolbox: Stage.Types.Toolbox;
 }
 
-const RemoveTokenButton = ({ tokenId, toolbox }: RemoveTokenButtonProps) => {
+const RemoveTokenButton = ({ token, toolbox }: RemoveTokenButtonProps) => {
     const [isModalVisible, showModal, hideModal] = useBoolean();
     const [deletingStatus, setDeletingStatus] = useState<RequestStatus>(RequestStatus.INITIAL);
+    const deleteModalContent = useMemo(() => {
+        const translationSufix = token.description ? 'withDescription' : 'withoutDescription';
```
Comment (qooban) [inline widgets/tokens/src/RemoveTokenButton.tsx:22]: Minor thing 🙂 

```suggestion
        const translationSuffix = token.description ? 'withDescription' : 'withoutDescription';
```

**Review by qooban — APPROVED** [review]
Didn't manage to test the UI, but code looks good. Added just one minor comment.

File: widgets/tokens/src/RemoveTokenButton.tsx (hunk 1 above)
Comment (NorbertAmartus) [reply widgets/tokens/src/RemoveTokenButton.tsx:22]: Minor, but quite accurate! 😁 
Thanks for an eagle eye! 🦅

**Review by qooban — APPROVED** [review]
Let's merge it! 🚀 

... and let it blow others minds over the long weekend 💥😄

**Comment by NorbertAmartus** [comment]
>Let's merge it! 🚀
> ... and let it blow others minds over the long weekend 💥😄

Seems like a reasonable idea 😄 🚀

**Pull request merged**